workshop/content/exercises/create-pipeline.adoc

```diff
@@ -42,29 +42,41 @@ spec:
+You might have noticed that there are no references to the PetClinic Git repository and its image in the registry. That's because `Pipelines` in Tekton are designed to be generic and reusable across environments and stages through the application's lifecycle. `Pipelines` abstract away the specifics of the Git source repository and image to be produced as `pipelineresources`. When triggering a `pipeline`, you can provide different Git repositories and image registries to be used during a `pipeline` execution. You will do that in a little bit in the next section.
 
-The tasks execution order is determined based on the dependencies that are defined between the tasks via `inputs` and `outputs` as well as explicit orders that are defined via `runAfter`.
+The `tasks` execution order is determined based on the dependencies that are defined between the `tasks` via `inputs` and `outputs` as well as explicit orders that are defined via `runAfter`. You'll notice the `deploy` task above has a `runAfter` specifying to only execute after the `build` task is complete.
 
-Save the pipeline definition as a `.yaml` file and then create it using `oc create -f YOUR_FILE_NAME.yaml`. Alternatively, in the OpenShift console, you can click on **Add &#8594; Import YAML** at the top right of the screen while you are in the **pipelines-tutorial** project, paste the YAML into the textfield, and click on **Create**.
+To create the `pipeline` above, you will have the option of using the OpenShift CLI (`oc`) or using the OpenShift Web Console:
+
+= OpenShift CLI
+
+The command below uses `oc` to take the `pipeline` definition from above from a local directory and then creates it in your OpenShift project. Run the command to create the `pipeline`:
+
+[source,bash,role=execute-1]
+----
+oc create -f ./code/exercise/deploy-pipeline.yaml
+----
+
+= OpenShift Web Console
+
+Alternatively, in the OpenShift web console, you can click on **Add &#8594; Import YAML** at the top right of the screen while you are in the **pipelines-tutorial** project, paste the YAML into the textfield, and click on **Create**.
 
 image:../images/console-import-yaml-1.png[OpenShift Console - Import Yaml Menu]
 
 image:../images/console-import-yaml-2.png[OpenShift Console - Import Yaml]
 
+You can check the list of `pipelines` you have created using the [Tekton CLI](https://github.com/tektoncd/cli/releases)(`tkn`):
 
-Check the list of pipelines you have created using the CLI:
-
-[source,bash]
+[source,bash,role=execute-1]
 ----
 $ tkn pipeline ls
 NAME             AGE              LAST RUN   STARTED   DURATION   STATUS
-build-pipeline   25 seconds ago   ---        ---       ---        ---
+deploy-pipeline  25 seconds ago   ---        ---       ---        ---
```

Comment:
This is actually called `deploy-pipeline` and is a typo that I corrected in the OpenShift pipelines tutorial.